packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.9.4
```

Comment:
This needs a new version bump; the previous one was lost in a merge.

Reply:
The issue has been fixed. A new version bump has been added.

Reply:
The package's actual version also needs to be bumped.

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8719859349019297185/+/u/Run_package_tests/CHANGELOG_and_version_validation/stdout